src/GafferScene/EditScopeAlgo.cpp

```diff
@@ -73,6 +73,19 @@ using namespace GafferScene;
 namespace
 {
 
+/// \todo Make this into an API for nodes to register the attributes and corresponding
+/// default values they will make available. This is needed to get the value of an
```

Comment:
I was thinking that the registry would be independent of nodes, and nodes would query the registry rather than populate it. So for instance, `startup/GafferArnold/attributes.py` might configure the attributes, and then ArnoldAttributes would just query them and generate the plugs automatically.

Reply:
Fixed in 4a64f56a1409f3ca0a909a0d91ffe9f2aa377b71